vignettes/tar_terra_tiles.Rmd

```diff
@@ -0,0 +1,386 @@
+---
+title: "Dynamic branching with raster tiles"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Dynamic branching with raster tiles}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+# With the root.dir option below,
+# this vignette runs the R code in a temporary directory
+# so new files are written to temporary storage
+# and not the user's file space.
+knitr::opts_knit$set(root.dir = fs::dir_create(tempfile()))
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.align = "center",
+  fig.height = 5,
+  fig.width = 5
+)
+if (identical(Sys.getenv("NOT_CRAN", unset = "false"), "false")) {
+  knitr::opts_chunk$set(eval = FALSE)
+}
```

Comment:
@njtierney, I just copied this from the other vignette you started—can you explain this?  It's the opposite of what I'd expect—eval gets set to FALSE when I build locally.

Reply:
I'm going to negate it for now, but let me know if I'm missing something

Reply:
Oh, I see now that `check()` sets `NOT_CRAN=true`, but maybe `build_vignettes()` doesn't? Anyway, this is back to how it was and should work now